### PR TITLE
Noah/assets

### PR DESCRIPTION
**Pull request by noahtsuei**

### CODE REVIEW

File: src/assets/ebike_data.js [hunk 1]
```diff
@@ -223,6 +223,134 @@ export const data = {
...
+      throttle_control: 'thumb',
+      crank_set_teeth: 46
+    },
+    {
+      id: 11,
+      archive: false,
+      brand: 'Aventon',
+      model: 'Level Commuter',
+      class: 3,
+      style: ['thin tire', 'front suspension', 'urban',''],
```
Comment (zhanghorace1) [inline src/assets/ebike_data.js:361]: Please remove 'thin tire', and the empty quotes at the end of the array

File: src/assets/ebike_data.js [hunk 2]
```diff
@@ -223,6 +223,134 @@ export const data = {
...
+      crank_set_teeth: 46
+    },
+    {
+      id: 10,
+      archive: false,
+      brand: 'Aventon',
+      model: 'Level.2 Commuter Step-Through',
+      class: 3,
+      style: ['thin tire', 'front suspension', 'urban',''],
+      accessories: ['basket', 'fenders', 'front rack', 'head light', 'rear rack','aventon battery',],
```
Comment (zhanghorace1) [inline src/assets/ebike_data.js]: Please remove the extra comma after 'aventon battery'

File: src/assets/ebike_data.js [hunk 3]
```diff
@@ -223,6 +223,134 @@ export const data = {
...
+      controller_a: -1,
+      motor_location: 'hub drive',
+      motor_wattage: 750,
+      motor_brand: 'No Data',
+      sensor: 'cadence',
+      torque_max_nm: -1,
+      range: 40,
+      brake_type: 'hydraulic',
+      speeds: 8,
+      image_url: ['https://cdn.shopify.com/s/files/1/1520/2468/products/2020_Aventon_Level_StoneGrey_StudioPhotos_001.jpg?v=1649284315&width=1200'],
```
Comment (zhanghorace1) [inline src/assets/ebike_data.js:382]: Please add 2 more image URLs to this array

File: src/assets/ebike_data.js [hunk 4]
```diff
@@ -223,6 +223,165 @@ export const data = {
...
+      crank_set_teeth: 46
+    },
+    {
+      id: 9,
+      archive: false,
+      brand: 'Aventon',
+      model: 'Level.2 Commuter',
+      class: 3,
+      style: ['thin tire', 'front suspension', 'urban',],
+      accessories: ['basket', 'fenders', 'front rack', 'head light', 'rear rack','aventon battery',],
```
Comment (zhanghorace1) [inline src/assets/ebike_data.js:266]: Please remove the extra comma after 'aventon battery'

File: src/assets/ebike_data.js [hunk 5]
```diff
@@ -223,6 +223,165 @@ export const data = {
...
+      throttle_control: 'thumb',
+      crank_set_teeth: 46
+    },
+    {
+      id: 12,
+      archive: false,
+      brand: 'Aventon',
+      model: 'Level Commuter',
+      class: 3,
+      style: ['thin tire', 'front suspension', 'urban',''],
```
Comment (zhanghorace1) [inline src/assets/ebike_data.js:361]: Please remove 'thin tire', and the empty quotes at the end of the array

File: src/assets/ebike_data.js [hunk 6]
```diff
@@ -223,6 +223,165 @@ export const data = {
...
+      accessories: ['basket', 'fenders', 'front rack', 'head light', 'rear rack',],
```
Comment (zhanghorace1) [inline src/assets/ebike_data.js:362]: Please remove the extra comma after 'rear rack'

File: src/assets/ebike_data.js [hunk 7]
```diff
@@ -223,6 +223,165 @@ export const data = {
...
+      controller_a: -1,
+      motor_location: 'hub drive',
+      motor_wattage: 750,
+      motor_brand: 'No Data',
+      sensor: 'cadence',
+      torque_max_nm: -1,
+      range: 40,
+      brake_type: 'hydraulic',
+      speeds: 8,
+      image_url: ['https://cdn.shopify.com/s/files/1/1520/2468/products/2020_Aventon_Level_StoneGrey_StudioPhotos_001.jpg?v=1649284315&width=1200'],
```
Comment (zhanghorace1) [inline src/assets/ebike_data.js:382]: Please add 2 more image URLs to this array

Comment (noahtsuei) [reply src/assets/ebike_data.js:382]: there is only one color 

can u tell me what parts of the bike images to send

**Pull request merged**